src/issuer.ts

```diff
@@ -1,7 +1,7 @@
 import { JWTHeaderParameters, SignJWT } from 'jose';
-import { packSDJWT } from './common';
-import { FORMAT_SEPARATOR, SD_JWT_TYPE } from './constants';
-import { IssueSDJWT } from './types';
+import { packSDJWT } from './common.js';
```

Comment:
@ragnika why do you suddenly need `.js` extension everywhere? Repository itself still seems to be in `commonjs`.

Reply:
It wasn't allowing me to build the esm package otherwise. I'll have to double check if that was because i accidentally left "type": "module" while i was testing it